examples/chip-tool/templates/logging/DataModelLogger-src.zapt

```diff
@@ -122,7 +124,7 @@ CHIP_ERROR DataModelLogger::LogCommand(const chip::app::ConcreteCommandPath & pa
             case {{asUpperCamelCase parent.name}}::Commands::{{asUpperCamelCase name}}::Id:
             {
                 {{asUpperCamelCase parent.name}}::Commands::{{asUpperCamelCase name}}::DecodableType value;
-                ReturnErrorOnFailure(chip::app::DataModel::Decode(*data, value));
+                ReturnErrorOnFailure(chip::app::DataModel::Decode(*data, value, kUndefinedFabricIndex));
```

Comment:
Is this logging incoming or outgoing commands?

Reply:
🤷‍♂️😬 how do I find out?

Reply:
Does seem to be client-side & unneeded after last change